packages/core/src/auth.ts

```diff
@@ -16,6 +16,9 @@ export enum AuthPurpose {
     GetLegacyData = "get_legacy_data",
     AccessKeyStore = "access_key_store",
     TestAuthenticator = "test_authenticator",
+    v3_Signup = 0,
+    v3_Login = 1,
+    v3_Recover = 2,
```

Comment:
I defined them here. I didn't define the `GetLegacyData` as well, but I believe I should, right?